extensions/Lily/HackedBlocks.js

```diff
@@ -26,38 +29,37 @@
             xml: '<block id="while" type="control_while"/>',

```

Comment:
I think you're right in that we should remove "while", I left it in by mistake while testing the extension. It doesn't need to be here.

Reply:
Honestly I think we should remove `while` from the regular Control category and keep it in this extension. I've never used it and it is the only block that has been changed in the entire default block palette now that `for` is gone.

Reply:
People were putting in issues when the "for" loop was removed, claiming it was a bug (and that was necessary to fix some strange behaviours).

Removing "while" from the main palette, while certainly the desirable option, is not a good idea

Reply:
Being one of those people, I was annoyed that there was no way to get the for loop back if I still needed it other than searching for an existing project that has the block or hacking it in myself.

With this, we only need to mention that the while loop has been moved to an extension category and you can easily import it from the extension gallery.